### ISSUE TITLE
label-spacing

### ISSUE THREAD

**Issue by MunMunMiao** [vuejs/eslint-plugin-vue#897]
**Please describe what the rule should do:**
Increase readability by adding tag spacing

**What category should the rule belong to?**
- [X] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

* close```{before: false, after: false}```
```vue
<!-- bad -->
<template>
    <div class="header"></div>
    <div class="content"></div>
    <div class="footer"></div>
</template>
```
* open```{before: true, after: true}```
```vue
<!-- good -->
<template>
    <!-- spacing -->
    <div class="header"></div>
    <!-- spacing -->
    <div class="content"></div>
    <!-- spacing -->
    <div class="footer"></div>
    <!-- spacing -->
</template>
```
* only before```{before: true, after: false}```
```vue
<template>
    <!-- spacing -->
    <div class="header"></div>
    <!-- spacing -->
    <div class="content"></div>
    <!-- spacing -->
    <div class="footer"></div>
</template>
```
* only after```{before: false, after: true}```
```vue
<template>
    <div class="header"></div>
    <!-- spacing -->
    <div class="content"></div>
    <!-- spacing -->
    <div class="footer"></div>
    <!-- spacing -->
</template>
```
* more examples
```vue
<template>
    <!-- spacing -->
    <div class="header">
        <!-- spacing -->
        <div class="logo"></div>
        <!-- spacing -->
    </div>
    <!-- spacing -->
    <div class="content">
        <!-- spacing -->
        <div class="left"></div>
        <!-- spacing -->
        <div class="right"></div>
        <!-- spacing -->
    </div>
    <!-- spacing -->
    <div class="footer">
        <!-- spacing -->
        <span>XXXXX(at)gmail.com</span>
        <!-- spacing -->
    </div>
    <!-- spacing -->
</template>
```

**Additional context**
